Binary_search/Binary_search.py

```diff
@@ -0,0 +1,39 @@
+# Binary search task A_2
+# https://informatics.msk.ru/mod/statements/view.php?id=192
+
+'''
+n, k = [int(i) for i in input().split()]
+
+N = [int(i) for i in input().split()]
+K = [int(i) for i in input().split()]
+'''
+
+def Bin_search(Arr, numberToSearch, leftBoard, rightBoard):

```

Comment:
All functions should named in lowercase letters.
Also, you always pass `left` and `right`. You don't really need them. You can extract this information from the array itself. `left = 0`, `right = len(array) - 1`.
```suggestion
def bin_search(array, number):
```